roles/tarzan/tasks/main.yml

```diff
@@ -16,9 +16,20 @@
               backup=no
   tags: tarzan
 
+- name: add the lua_path to funk_004's environment via the sourceme_upps script
+  lineinfile: dest={{ ngi_pipeline_conf }}/{{ bash_env_upps_script }}
+              line='export LUA_PATH={{ lua_path }}'
+              backup=no
```

Comment:
If we really want to go this route, wouldn't it make more sense for ngi_pipeline and charon to add their respective paths in the same way?